types.test.ts

```diff
@@ -20,5 +20,5 @@ app.register(fastifyAuth).after((err) => {
       done(new Error());
       done();
     },
-  ]);
+  ], {relation: 'or'});
```

Comment:
I think we should check both APIs, with and without the options

Reply:
Makes sense. Should i simply add it as another call to `app.auth` like this:

```ts
app.register(fastifyAuth).after((err) => {
  app.auth([
    (
      _request: fastify.FastifyRequest<
        http.IncomingMessage,
        fastify.DefaultQuery,
        fastify.DefaultParams,
        fastify.DefaultHeaders,
        any
      >,
      _reply: fastify.FastifyReply<http.ServerResponse>,
      done
    ) => {
      done(new Error());
      done();
    },
  ], {relation: 'or'});
  app.auth([
    (
      _request: fastify.FastifyRequest<
        http.IncomingMessage,
        fastify.DefaultQuery,
        fastify.DefaultParams,
        fastify.DefaultHeaders,
        any
      >,
      _reply: fastify.FastifyReply<http.ServerResponse>,
      done
    ) => {
      done(new Error());
      done();
    },
  ]);
});
```
or do you want it somewhere else?

Reply:
That example is ok